cinema/views.py

```diff
@@ -56,3 +90,39 @@
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        date = self.request.query_params.get("date")
+        movies = self.request.query_params.get("movie")
+
+        queryset = self.queryset
+
+        if date:
+            queryset = queryset.filter(show_time__date=date)
+
+        if movies:
+            movies_ids = [int(str_id) for str_id in movies.split(",")]
+            queryset = queryset.filter(movie__id__in=movies_ids)
```

Comment:
The filtering logic for movies should be combined using the `queryset` variable instead of reassigning it to a new query. This ensures that all filters are applied cumulatively.